components/About/index.js

```diff
@@ -4,7 +4,7 @@ const About = ({ data }) => {
   const { heading, text, image } = data;
   return (
     <div
-      data-cy="who-am-i"
+      data-test-id="about"
```

Comment:
Changed to `about` so that we use the same string in the Playwright test
```
  await page.getByRole('link', { name: 'About me' }).click();
  await page.waitForURL('/#about');
  await expect(page.getByTestId('about')).toBeInViewport();
```